### ISSUE TITLE
Landing page idea

### ISSUE THREAD

**Issue by chigy** [chigy/winui-redline#13]
The idea I had was to put all the controls in the rest state (I do have some items that has popup open but don't have to maybe) so people can see them in one place. Maybe this needs to be more of a vertically scrolling page than horizontally long page like this, though.

@kikisaints FYI

![ControlsSheet](https://user-images.githubusercontent.com/22958680/61746206-0d154f00-ad50-11e9-916e-1a9a2cfd7fcc.png)

**Comment by kikisaints** [comment]
Were you imagining that the user would click on one of those controls and it would take them to the redlines page?

**Comment by chigy** [comment]
Yes, also was thinking maybe hover over the control and it tells you the name of the control or something like that.
